app/connectors/FinancialDetailsConnector.scala

```diff
@@ -90,6 +98,31 @@ class FinancialDetailsConnector @Inject()(val http: HttpClient,
 
   }
 
+  def getCreditsAndRefund(taxYear: Int, nino: String)
```

Comment:
* we should use TaxYear type in all of the cases were possible;
* dateFrom and dateTo should be accessible as attributes from taxYear value